src/components/UpdateUser.js

```diff
@@ -0,0 +1,26 @@
+import { useState } from "react";
+
+const UpdateUser = ( { user, token, setUser, } ) => {
+    const [username, setUsername] = useState();
+    const [email, setEmail] = useState();
+    const [password, setPassword] = useState();
+
+    const submitHandler = (e) => {
+        e.preventDefault();
+        console.log("update user - ", username, email, password);
+        console.log("ToDo: UpdateUser -> updateUser( {user}, token, setUser )")
+    }
+
+    return(
+        <div>
+            <form onSubmit={submitHandler}>
+                <input onChange={(e) => setUsername(e.target.value)} />
```

Comment:
Sorry mate I removed it because I thought you said that typically; a user wont be able to change a username, rather they will only be able to change their personal details. I've added username back into it : )